packages/odyssey-design-tokens/src/color/background.json

```diff
@@ -0,0 +1,28 @@
+{
+  "color": {
+    "background": {
+      "base": { "value": "{color.palette.neutral.white.value}" },
+      "disabled": { "value": "{color.palette.neutral.000.value}" },
+      "primary": {
+        "light": { "value": "{color.palette.blue.000.value}" },
+        "base": { "value": "{color.palette.blue.300.value}" },
+        "dark": { "value": "{color.palette.blue.900.value}" }
+      },
+      "danger": {
+        "light": { "value": "{color.palette.red.000.value}" },
+        "base": { "value": "{color.palette.red.300.value}" },
+        "dark": { "value": "{color.palette.red.500.value}" }
+      },
+      "caution": {
+        "light": { "value": "{color.palette.yellow.000.value}" },
+        "base": { "value": "{color.palette.yellow.300.value}" },
+        "dark": { "value": "{color.palette.yellow.500.value}" }
+      },
+      "success": {
+        "light": { "value": "{color.palette.green.000.value}" },
+        "base": { "value": "{color.palette.green.300.value}" },
+        "dark": { "value": "{color.palette.green.600.value}" }
+      }
```

Comment:
@josesolano-okta @okta/design-system I was originally sad to see that these background tints/shades do not align 100% with the functional color tints/shades even though they share the same naming structure: `PrimaryLight` or `SuccessDark` etc. I think perhaps this is OK, but I worry about the confusion of presenting functional colors that are not used in practice. It may be better to move those functional colors into more specific groupings to denote the fact that they are not universally applied in all `Primary` or `Success` cases?

Reply:
AFAIK we don't have a standardized set of "background" color tokens yet, which is why there's variance across components.

Box deploys these, but only because we issues a catch-all on property application. They might be better served as component-only variables within Box's .theme for now.

Reply:
@edburyenegren-okta the problem I'm calling out isn't an issue of variance across components specifically, but rather the fact that tokens with similar names contain different values. For example, `ColorPrimaryLight` references `blue300` and `ColorBackgroundPrimaryLight` references `blue000`. I would expect these to point to the same shade of blue given the names. Not a critical issue just unideal.

I know we haven't defined tokens on the design side yet for these but we need them on the eng side so lets keep these here and adjust as we move forward.